pkg/apis/discovery/fuzzer/fuzzer.go

```diff
@@ -32,7 +32,7 @@ var Funcs = func(codecs runtimeserializer.CodecFactory) []interface{} {
 
 			// match defaults
 			if obj.AddressType == nil {
-				ipAddressType := discovery.AddressTypeIP
+				ipAddressType := discovery.AddressTypeIPv4
```

Comment:
What this should do is pick a random value from supported set.

e.g. an example:

```
        func(p *core.Protocol, c fuzz.Continue) {                                                                          
            protocols := []core.Protocol{core.ProtocolTCP, core.ProtocolUDP, core.ProtocolSCTP}
            *p = protocols[c.Rand.Intn(len(protocols))]
        },
```

Reply:
Done, thanks!